### DISCUSSION TITLE
Invite all Followers (studio)

### DISCUSSION THREAD

**Discussion by Scratchfangs** [ScratchAddons/ScratchAddons#635] [Ideas] [answered]
What if there was a way on the addon to invite all of your followers with the advanced studio options?
This could really help people who has a lot of followers and they are inviting them to a studio.

**Comment by WorldLanguages** [comment, marked as answer]
No.

**Reply by WorldLanguages** [reply]
Just kidding - but still no. There are rate limits, and I don't think any type of automatic inviting would be ok.

**Comment by retronbv** [comment]
this would be horrible

**Comment by Explosion-Scratch** [comment]
Theoretically one can invite up to 200 people a day.

**Reply by retronbv** [reply]
> Theoretically one can invite up to 200 people a day.

that is still horrible

**Reply by just-a-github-user** [reply]
WHY is it horrible

**Reply by retronbv** [reply]
People already get mass studio invite spam. They don’t need a way for someone to spam invite people

**Reply by retronbv** [reply]
also: don’t necropost lol

**Reply by Melodie1** [reply]
it will be useful!

**Reply by retronbv** [reply]
> it will be useful!

i literally just told someone not to necropost……
